ports/atmel-samd/common-hal/busio/UART.c

```diff
@@ -76,6 +76,10 @@ void common_hal_busio_uart_construct(busio_uart_obj_t *self,
     if (bits > 8) {
         mp_raise_NotImplementedError(translate("bytes > 8 bits not supported"));
     }
+	
+	if (receiver_buffer_size % 4 != 0)	{
+		mp_raise_msg_varg(&mp_type_ValueError, translate("Invalid RX buffer size of %d, must be a multiple of 4"), receiver_buffer_size);
```

Comment:
This is a long message, and we need to save space. We don't need to print out the input argument, so I'd suggest this (using the argument name in `shared-bindings/busio/UART.c`):
```suggestion
		mp_raise_ValueErorr( translate("receiver_buffer_size must be a multiple of 4"), receiver_buffer_size);
```